index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
```

Comment:
While you did a good job working on the project. It would be nice to stick to the peoject requirements and design guidelines.

This is your design: 
![Screenshot from 2022-08-06 13-47-58](https://user-images.githubusercontent.com/79506105/183244558-1197a2f2-9afc-4589-bb3f-6b23fc5dbd9b.png)

![Screenshot from 2022-08-06 13-48-22](https://user-images.githubusercontent.com/79506105/183244585-1473ad27-6b6c-4987-86b0-e06db673262b.png)


![Screenshot from 2022-08-06 13-48-27](https://user-images.githubusercontent.com/79506105/183244597-faac0021-e8c9-455d-beba-5eb6f6d580e4.png)
![Screenshot from 2022-08-06 13-48-32](https://user-images.githubusercontent.com/79506105/183244612-a3e0d9df-c954-4ea8-a014-3487bbc950a5.png)
 
Please stick to this design:
![Screenshot from 2022-08-06 13-49-22](https://user-images.githubusercontent.com/79506105/183244629-c549082e-3ab3-45e0-887f-14025d43bec0.png)
![Screenshot from 2022-08-06 13-49-30](https://user-images.githu
![Screenshot from 2022-08-06 13-49-51](https://user-images.githubusercontent.com/79506105/183244636-fac83763-1167-4fb7-9e43-5bbdf9e7bf78.png)

![Screenshot from 2022-08-06 13-49-39](https://user-images.githubusercontent.com/79506105/183244633-8fa291fa-df61-402b-8b4d-5ad764ed7bcd.png)
busercontent.com/79506105/183244630-b5d4170d-16ba-4a2d-9bf3-9aa4ef703eda.png)

Reply:
Hello @KayLemba! thank you for your comment but I didn't really get you. what should I do to stick to the design? because the topic  I selected(web development platform) is different from the given design( conference). I already used the font based on the requirement.

Reply:
You have to follow the behance design look at the requirements properly

Reply:
@KayLemba  should I change everything or not? I selected all the background colors and images that are best for the topic I selected. I think that, if I use something from the conference page, it may not be good for the topic I selected.

Reply:
Yes change everything

Reply:
Please follow the guidelines on the project requirements.